packages/icons/saltIcons.css

```diff
@@ -0,0 +1,1106 @@
+/** WARNING: This file was generated by a script. Do not modify it manually */
+[class*=" saltIcon-"],
+[class^="saltIcon-"] {
+  mask-size: cover;
```

Comment:
Are you sure this is supported by all our supported browsers? From looking at caniuse.com it needs the webkit prefix.

Reply:
good point, will add it in